source/renderer/app/stores/ProfileStore.js

```diff
@@ -146,6 +152,17 @@ export default class SettingsStore extends Store {
     return this.getTermsOfUseAcceptanceRequest.result === true;
   }
 
+  @computed get hasLoadedDataLayerMigrationAcceptance(): boolean {
+    return (
+      this.getDataLayerMigrationAcceptanceRequest.wasExecuted &&
+      this.getDataLayerMigrationAcceptanceRequest.result !== null
+    );
+  }
+
+  @computed get hasDataLayerMigrationAcceptanceed(): boolean {
```

Comment:
@daniloprates you have a typo here:
`hasDataLayerMigrationAcceptanceed` -> `hasDataLayerMigrationAccepted`